packages/swapper/src/swappers/zrx/getZrxQuote/getZrxQuote.ts

```diff
@@ -33,12 +27,11 @@ export async function getZrxQuote(input: GetQuoteInput): Promise<Quote<ChainType
   const buyToken = buyAsset.tokenId || buyAsset.symbol
   const sellToken = sellAsset.tokenId || sellAsset.symbol
 
-  let minQuoteSellAmountWei = null
-  if (minQuoteSellAmount) {
-    minQuoteSellAmountWei = new BigNumber(minQuoteSellAmount as string).times(
-      new BigNumber(10).exponentiatedBy(sellAsset.precision)
-    )
-  }
+  const { minimum: minQuoteSellAmount, maximum: maxSellAmount } = await getZrxMinMax(input)
+
+  const minQuoteSellAmountWei = new BigNumber(minQuoteSellAmount).times(
```

Comment:
If we're changing to this let's make it bnorzero

Reply:
yep. done